### ISSUE TITLE
Interpreter returns an empty Blob for `ActorUrlE`

### ISSUE THREAD

**Issue by rvanasa** [dfinity/motoko#3683]
[Example](https://embed.smartcontracts.org/motoko/g/tRGPBbvb9rfpzMpD93yxK5DBPALTEuhuyTHbT1EajoN7sUPAMxK5ofnA64MbwKrUSC7Xzod52fmNo3mm65muo3xLLvnC3TifwG6tsHhpS?lines=6):
```motoko
let ic = actor "aaaaa-aa" : actor {
  raw_rand : () -> async Blob
}; // => "" : actor {raw_rand : shared () -> async Blob}

ic.raw_rand // Error; resolves to ("" : Blob).raw_rand
```

This currently prevents loading the `mo:base/Random` module in the interpreter. 

Would there be a relatively simple way to fix this? Otherwise, we could temporarily move the `raw_rand` declaration into the `Random.blob()` function to fix the initialization error (https://github.com/dfinity/motoko-base/pull/501). 

I briefly started working on a solution (using the actor type to create an object with placeholder functions) but wanted to check in about this before proceeding further.

**Comment by crusso** [comment]
See my comment https://github.com/dfinity/motoko-base/pull/501#pullrequestreview-1241918951

**Comment by rvanasa** [comment]
Fixed in #3693.
